cmd/skynet/skynetcmd.go

```diff
@@ -110,6 +157,75 @@ func skynetcmd() {
 	fmt.Printf("Default Skynet Portal: %v \n\n", downloadOptions.PortalURL)
 }
 
+// skynetaddskykey stores the given base-64 encoded skykey with the skykey
+// manager.
+func skynetaddskykeycmd(skykey string) {
+	// Get the addskykey options.
+	opts := skynet.DefaultAddSkykeyOptions
+	opts.Options = getCommonOptions(opts.Options)
+	fmt.Printf("AddSkykey Options: %+v\n", opts)
```

Comment:
Was this left over debug output? I don't think this needs to be printed every time. Might be helpful for debug if there is an error.

Reply:
Yeah, it was already there and I just copy/pasted into every function without thinking lol